specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/cloudService.json

```diff
@@ -0,0 +1,2680 @@
+    },
+    "CloudServiceVaultSecretGroup": {
+      "description": "Describes a set of certificates which are all in the same Key Vault.",
+      "type": "object",
+      "properties": {
+        "sourceVault": {
+          "$ref": "#/definitions/SubResource",
+          "description": "The relative URL of the Key Vault containing all of the certificates in VaultCertificates."
+        },
+        "vaultCertificates": {
+          "description": "The list of key vault references in SourceVault which contain certificates.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CloudServiceVaultCertificate"
+          }
+        }
+      }
+    },
+    "CloudServiceOsProfile": {
+      "description": "Describes the OS profile for the cloud service.",
+      "type": "object",
+      "properties": {
+        "secrets": {
+          "description": "Specifies set of certificates that should be installed onto the role instances.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CloudServiceVaultSecretGroup"
+          }
+        }
+      }
+    },
+    "LoadBalancerFrontendIPConfigurationProperties": {
+      "description": "Describes a cloud service IP Configuration",
+      "type": "object",
+      "properties": {
+        "publicIPAddress": {
+          "$ref": "#/definitions/SubResource",
+          "description": "The reference to the public ip address resource."
+        },
+        "subnet": {
+          "$ref": "#/definitions/SubResource",
+          "description": "The reference to the virtual network subnet resource."
+        },
+        "privateIPAddress": {
+          "description": "The virtual network private IP address of the IP configuration.",
+          "type": "string"
+        }
+      }
+    },
+    "LoadBalancerFrontendIPConfiguration": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the resource that is unique within the set of frontend IP configurations used by the load balancer. This name can be used to access the resource.",
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/LoadBalancerFrontendIPConfigurationProperties",
+          "description": "Properties of load balancer frontend ip configuration."
+        }
+      },
+      "required": [
+        "name",
+        "properties"
+      ]
+    },
+    "LoadBalancerConfigurationProperties": {
+      "type": "object",
+      "properties": {
+        "frontendIPConfigurations": {
+          "description": "Specifies the frontend IP to be used for the load balancer. Only IPv4 frontend IP address is supported. Each load balancer configuration must have exactly one frontend IP configuration.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LoadBalancerFrontendIPConfiguration"
+          }
+        }
+      },
+      "required": [
+        "frontendIPConfigurations"
```

Comment:
Making a previously optional property mandatory is a breaking change. 
Please remove the Required portion and make edits in the backend as necessary.

Reply:
taking below suggestion